src/Hl7.Fhir.Base/Model/Generated/Signature.cs

```diff
@@ -231,6 +263,38 @@ public byte[] Data
       }
     }
 
+    /// <summary>
+    /// The actual signature content (XML DigSig. JWS, picture, etc.). Note: Element is deprecated since R4, do not use with R4 and newer releases.
+    /// </summary>
+    [FhirElement("blob", Order=90)]
```

Comment:
Actually, blob was renamed to data -> much like in Binary. Shouldn't `data` say that it can be used since R4?

Reply:
Is it an idea to mention the word "rename" here? If so, we should probably do that for Binary too.

Reply:
For Binary I changed it now like this:
```
   /// The actual content. Note: Element is replaced by 'Binary.data' since R4. Do not use this element 'content' with R4 and newer releases.
```